packages/mobile/src/components/edit/PriceAndAudienceField/PriceAndAudienceScreen.tsx

```diff
@@ -168,30 +168,20 @@ export const PriceAndAudienceScreen = () => {
   const handleSubmit = useCallback(() => {
     validateForm() // Fixes any erroneous errors that haven't been revalidated
     if (!isUpload && isEditableAccessEnabled && usersMayLoseAccess) {
-      dispatch(
-        modalsActions.setVisibility({
-          modal: 'EditPriceAndAudienceConfirmation',
-          visible: true
-        })
-      )
+      onOpenEditAccessConfirmationModal({
+        confirmCallback: navigation.goBack,
```

Comment:
shouldn't this take action on confirm?

Reply:
it doesn't need to because of how these forms are written currently (unfortunately). any action you take here mutates the form state, so this is kind of just informative.